doc/source/getting_started.rst

```diff
@@ -0,0 +1,388 @@
+    Don’t panic! The solution is then to precise to which axis belong the labels
+    using the special notation ``x.axis_name``.
+
+.. ipython:: python
+   :doctest:
+
+    arr2 = ndrange('a=0..3;b=0..3')
+    arr2
+
+    arr2[x.a['0,2'], x.b['1:3']]
+
+Using positions
+^^^^^^^^^^^^^^^
+
+Sometimes it is more practical to use positions along the axis, instead of labels.
+You need to add the character ``i`` before the brackets: ``.i[positions]``.
+Positions can be also negative (-1 represent the last element of an axis).
+
+.. warning::
+    * Remember that positions (indices) are always 0-based in Python.
+      So the first element is at position 0, the second is at position 1, etc.
+    * The `end` argument of a slice is EXCLUSIVE in context of positions (``1:4 = [1, 2, 3]``)
+      while it is INCLUSIVE in context of labels (``'a0:a2' = 'a0,a1,a2'``).
+
+.. ipython:: python
+   :doctest:
+
+    arr
+
+    # select label at position 0 of first axis and label at position 1 of second axis
+    arr.i[0, 1]
+
+    # or equivalently
+    arr['a0', 'b1']
+
+    # first row
+    arr.i[0]
+
+    # or equivalently
+    arr['a0']
+
+    # first column
+    # `:` will select the entire axis
+    arr.i[:, 0]
+
+    # or equivalently
+    arr['b0']
+
+    # when using slices in the context of selecting by position,
+    # the end argument is EXCLUSIVE (as usual in Python): 1:4 stands for [1, 2, 3]
+    arr.i[[0, 2], 1:4]
+
+    # or equivalently
+    arr['a0,a2', 'b1:b3']
+
+The notation ``.i[positions]`` can be also used on axes:
+
+.. ipython:: python
+   :doctest:
+
+    arr[x.a.i[0, 2], x.b.i[1:4]]
+
+    # or equivalently
+    arr['a0,a2', 'b1:b3']
+
+Aggregation
+-----------
+
+The LArray library includes many aggregations methods.
+For example, to calculate the sum along an axis, write:
+
+.. ipython:: python
+   :doctest:
+
+    arr.sum('a')
+
+    # or equivalently
+    arr.sum(x.a)
+
+To aggregate along all axes except one, you simply have to append `_by`
+to the aggregation method you want to use:
+
+.. ipython:: python
+   :doctest:
+
+    arr.sum_by('a')
+
+    # or equivalently
+    arr.sum_by(x.a)
+
+See :ref:`here <la_agg>` to get the list of all available aggregation methods.
+
+Groups
+------
+
+A :ref:`Group <api-group>` represents a subset of labels or positions of an axis:
+
+.. ipython:: python
+   :doctest:
+
+    even = arr.a['a0::2']
+    even
+
+    odd = arr.a['a1::2']
+    odd
+
+They can be used in selections:
+
+.. ipython:: python
+   :doctest:
+
+    arr[even]
+
+    arr[odd]
+
+or aggregations:
+
+.. ipython:: python
+   :doctest:
+
+    arr.sum((even, odd))
+
+In the case of aggregations, it is often useful to attach them a name
+using the operator ``>>``:
+
+.. ipython:: python
+   :doctest:
+
+    arr.sum((even >> 'even', odd >> 'odd'))
+
+    # groups can also be created "in-flight"
+    arr.sum('a0::2 >> even; a1::2 >> odd')
+
+Compatibility with pandas
```

Comment:
As much as I would like LArray to be widely used by the Python community (personally, the more people use my work, the more satisfied I am), I am unsure we will ever get to that point. This is for several reasons:
* our current users push us in a direction which is unusual to the wider Python community (guess axis, string shortcut syntaxes) and also unusual for the Python scientific community (not web/notebook based).
* we currently do not have the time to communicate around this project, or more generally build a community, and I do not see this changing anytime soon.
* I would have trouble selling to our management that investing (a significant part of our time) into building a community would be worth it. Yes it might be worth it in the long run, but the chances are slim to ever pay off our investment.

Yes, I am frustrated by the current situation, but I do not have any solution. If you have I am all hears ;-).